macros/pivot_event_properties_json.sql

```diff
@@ -9,6 +9,9 @@ replace(parse_json(event_properties):"{{ property }}", '"', '')
 
 {%- elif target.type == 'redshift' -%}
 replace(json_extract_path_text(event_properties, {{ "'" ~ property ~ "'" }} ), '"', '')
+
+{%- elif target.type == 'postgres' -%}
+replace(json_extract_path_text(event_properties, {{ "'" ~ property ~ "'" }} ), '"', '')
```

Comment:
Can we do either of the below instead?
- Combine the redshift and postgres conditional to be `{%- elif target.type in ('redshift', 'postgres') -%}
- Is this a macro that our [current json_extract](https://github.com/fivetran/dbt_fivetran_utils/blob/master/macros/json_extract.sql) could handle? Or should we consider moving this macro to our dbt_fivetran_utils?

Reply:
ah that's a good call out -- i've always felt like this belongs in fivetran_utils so i'll clean it up + move it there